docs/TFM/welcome.md

```diff
@@ -1,2 +1,11 @@
 # Tools, Flows & Methodologies
 Here we will describe the Tools, Flows & methodologies (TFM) of this project.  
+We strongly recommend that you have a solid understanding of the foundational topics outlined in the TFM section.
+
+Please make sure to follow the subjects in the left side bar and install the necessary software. Keep in mind that some topics are mandatory, such as software installation, while others provide additional information that may be useful in the future.
+
+Feel free to contact us through the [Contact Us](/docs/contact_us/contact.md) page if you have any questions.
```

Comment:
I think The contact information should map to the github discussion or the gihub issues